README.md

```
@@ -1,96 +1,86 @@
-![Components](Add photo of your components here)
-*List out all components shown*
+[LED](https://github.com/a-nuranj/anuranj/blob/main/WhatsApp%20Image%202025-08-09%20at%2005.19.09_6af1bd64.jpg)
+[Resistor](https://github.com/a-nuranj/anuranj/blob/main/WhatsApp%20Image%202025-08-09%20at%2005.19.10_dc262e0a.jpg)
+[esp32 WROOM](http://github.com/a-nuranj/anuranj/blob/main/WhatsApp%20Image%202025-08-09%20at%2005.19.10_af895e05.jpg)
 
-![Build](Add photos of build process here)
-*Explain the build steps*
-
-![Final](Add photo of final product here)
-*Explain the final build*
+!Final](https://github.com/a-nuranj/anuranj/blob/main/WhatsApp%20Image%202025-08-09%20at%2005.10.22_b8beafa9.jpg)
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)